src/components/InstructorLessonsCard/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import Card from 'components/Card'
-import LessonListsByStates from 'components/LessonListsByStates'
+import LessonGroups from 'components/LessonGroups'
 
 export default ({instructor}) => (
   <Card title={`${instructor.first_name}'s lessons`}>
-    <LessonListsByStates instructor={instructor} />
+    <LessonGroups instructor={instructor} />
   </Card>
 )
```

Comment:
Simpler name for this component used in multiple parts of the app:
![image](https://cloud.githubusercontent.com/assets/5497885/24067043/6d57e916-0b3d-11e7-94d3-cb93357d8a65.png)